README.md

```diff
@@ -16,3 +16,9 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+The way i would initially go about verifying if it is a $O(n)$ runtime is to graph it. If i graphed it a bunch of values untill it started running too slow i would get a pretty good idea of its runtime. I would at the same time graph a known sorting algorithm such as Quicksort and insertion sort. I would do Quick sort because it generally runs the fastest and insertion sort because best case it is also $O(n)$. I would make a script that keeps running larger and larger input sizes untill it takes longer than 10 min to run on my machine for each algorithm. This way by the end i can also compare the size of n that each algorithm got to. Once i get my results and make my graph i would look at it and if the mystery algorithm is close to being a straight line i would believe them. i would also want for it to be the fastest algorithm for sorting.
+
+For a theoretical argument as to why X could not be correct, I recall in class you talking about how a sorting algorithm of time complexity $O(n)$ shouldnt be possible becase that doesnt leave for any room to actually compare any number of elements to any other elements.
```

Comment:
And more specifically, what is the complexity of the comparison-based sorting problem?